CTS1.rst

```diff
@@ -0,0 +1,666 @@
+
+  {
+    "description": "CTS1 indpakket GTU"
+  }
+
+Example token metadata: Non-fungible
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An example of token metadata for a NFT could be:
+
+.. code-block:: json
+
+  {
+    "name": "Bibi - The Ryan Cat",
+    "description": "Ryan cats are lonely creatures travelling the galaxy in search of their ancestors and true inheritance",
+    "thumbnail": { "url": "https://location.of/the/thumbnail.png" },
+    "display": { "url": "https://location.of/the/display.png" },
+    "attributes": [{
+      "type": "date",
+      "name": "Birthday",
+      "value": "1629792199610"
+    }, {
+      "type": "string",
+      "name": "Body",
+      "value": "Strong"
+    }, {
+      "type": "string",
+      "name": "Head",
+      "value": "Round"
+    }, {
+      "type": "string",
+      "name": "Tail",
+      "value": "Short"
+    }],
+    "localization": {
+      "da-DK": {
+        "url": "https://location.of/the/danish/metadata.json",
+        "hash": "588d7c14883231cfee522479cc66565fd9a50024603a7b8c99bd7869ca2f0ea3"
+      }
+    }
+  }
+
+The danish localization JSON file could be:
+
+.. code-block:: json
+
+  {
+    "name": "Bibi - Ryan katten",
+    "description": "Ryan katte er ensomme væsner, som rejser rundt i galaxen søgende efter deres forfædre og sande fortid"
+  }
+
+Smart contract limitations
+==========================
+
+A number of limitations are important to be aware of:
+
+- Smart contract state size is limited to 16 KiB.
+- Smart contract function parameters are limited to 1 KiB.
+- Each logged event is limited to 0.5 KiB.
+- The number of logged events is limited to 64.
+- The total size of the smart contract module is limited to 64 KiB.
+
+.. note::
+
+  Smart contracts where the contract state size limit is to low, can distribute the state across multiple smart contract instances.
+
+Decisions and rationale
+=======================
+
+In this section we point out some of the differences from other popular token standards found on other blockchains, and provide reasons for deviating from them in CTS1.
+
+Token ID bytes instead an integer
+---------------------------------
+
+Token standards such as ERC721 and ERC1155 both uses an 256 bit unsigned integer (32 bytes) for the token ID, to support using something like a SHA256 hash for the token ID.
+But in the case where the token ID have no significance other than a simple identifier, smaller sized token IDs can reduce energy costs.
+This is why we chose to let the first byte indicate the size of the token ID, meaning a token ID can vary between 1 byte and 256 bytes, resulting in more than 10^614 possible token IDs.
+
+Only batched transfers
+----------------------
+
+The specification only has a ``transfer`` smart contract function which takes list of transfer and no function for a single transfer.
+This will result in lower energy cost compared to multiple contract calls and only introduce a small overhead for single transfers.
```

Comment:
```suggestion
This will result in lower energy cost compared to multiple contract calls and only introduces a small overhead for single transfers.
```
```